src/bookmarks-db.js

```diff
@@ -189,6 +189,22 @@ export async function getBookmarks(limit = 10, offset = 0) {
   return undefined;
 }
 
+export async function getBookmarksForCSVExport() {
+  // We use a try catch block in case of db errors
+  try {
+    const results = await db.all(
+      'SELECT title,url,description,tags,created_at,updated_at from bookmarks',
+    );
+    // These titles should be updated if the query changes above.
+    const columnTitles = {'title': 'title','url': 'url','description': 'description','tags': 'tags','created_at': 'created_at','updated_at': 'updated_at'};
```

Comment:
csv-stringify does have a way to specify titles when it stringifies, but I wanted to keep the titles local to this function so people didn't forget to update them if the query changed.

Reply:
I think we could get a little clever here and remove the need to update two things: something like this?

```
const columns = ['title', 'url', 'description', 'tags', 'created_at', 'updated_at'];
const query = `SELECT ${columns.join(',')} from bookmarks`;
const columnTitles = columns.reduce((o, c) => { o[c] = c; return o}, {})
```

Reply:
(that probably doesn't pass eslint, apologies, haha)